src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -112,5 +112,6 @@ private Error() {
         public static final String AUTHENTICATION_TOKEN_INVALID =
             "Authentication failure due to invalid / expired tokens (IDAM / S2S).";
         public static final String UNAUTHORISED_S2S_SERVICE = "Unauthorised S2S service";
+        public static final String CASE_TYPE_INVALID = "Case Type is not valid";
```

Comment:
"Case Type is not valid" --> "Case Type Id is not valid."
(This should be reflected to functional test response data)